src/main/java/step3/Main.java

```diff
@@ -0,0 +1,14 @@
+package step3;
+
+import step3.controller.RacingCarGame;
+import step3.view.InputView;
+import step3.view.ResultView;
+
+public class Main {
+
+    public static void main(String[] args) {
+        InputView inputView = new InputView();
+        RacingCarGame racingCarGame = new RacingCarGame(inputView.inputCar(), inputView.inputAttempt());
+        new ResultView().print(racingCarGame);
```

Comment:
ResultView 에서 게임을 실행하지 않고
실행된 결과를 출력해주도록 변경이 되면 어떨까요 :) ?
```suggestion
        RacingCarGame racingCarGame = new RacingCarGame(inputView.inputCar(), inputView.inputAttempt());
        new ResultView().print(racingCarGame.start());
```